pkg/runtime/tags.go

```diff
@@ -23,19 +22,59 @@ import (
 	acktypes "github.com/aws-controllers-k8s/runtime/pkg/types"
 )
 
+// expandTagFormat is a function which returns the resolved value for an
+// ACK resource tag format. Ex: %CONTROLLER_SERVICE% -> s3
+type expandTagFormat func(rtclient.Object, acktypes.ServiceControllerMetadata) string
```

Comment:
I am also not super convinced with the name `expandTagFormat`, maybe something like `resolveTagValues` makes more sense?

Reply:
changed it to `resolveTagFormat`